libff/algebra/field_utils/algorithms.tcc

```diff
@@ -47,6 +47,58 @@ FieldT power(const FieldT &base, const unsigned long exponent)
     return power<FieldT>(base, bigint<1>(exponent));
```

Comment:
Can the power with an `unsigned long` call the power with an `unsigned long long` instead of the bigint power?